lib/curl_config.h.cmake

```diff
@@ -770,6 +770,9 @@ ${SIZEOF_TIME_T_CODE}
 /* to enable Windows SSL  */
 #cmakedefine USE_SCHANNEL 1
 
+/* to enable Windows CNG APIs available in Vista and above */
+#cmakedefine HAVE_NCRYPT 1
```

Comment:
The next release will require Vista, and ncrypt can be used unconditionally.
Let's delete these bits.